tokamak.py

```diff
@@ -13,6 +14,7 @@ def parse_args():
 	args = parser.parse_args()
 	return args
 
+
 def read_radial_build(file):
```

Comment:
Docstring might be useful for readers of the code.
https://realpython.com/documenting-python-code/#documenting-your-python-code-base-using-docstrings